pkg/plugin/pr-sanitizer/main.go

```diff
@@ -2,57 +2,42 @@ package main
 
 import (
 	gogh "github.com/google/go-github/github"
-	"github.com/sirupsen/logrus"
 	"k8s.io/test-infra/prow/pluginhelp"
 
 	"github.com/arquillian/ike-prow-plugins/pkg/github"
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
 	pluginBootstrap "github.com/arquillian/ike-prow-plugins/pkg/plugin"
 	"github.com/arquillian/ike-prow-plugins/pkg/plugin/server"
 )
 
 // ProwPluginName is an external prow plugin name used to register this service
 const ProwPluginName = "pr-sanitizer"
 
-var (
-	log = logrus.StandardLogger().WithField("ike-plugins", ProwPluginName)
-)
-
 // GitHubLabelsEventsHandler is the event handler for the plugin.
 // Implements server.GitHubEventHandler interface which contains the logic for incoming GitHub events
 type GitHubLabelsEventsHandler struct {
 	Client *gogh.Client
-	log    *logrus.Entry
 }
 
 func main() {
-	pluginBootstrap.InitPlugin(log, handlerCreator, serverCreator, helpProvider)
+	pluginBootstrap.InitPlugin(ProwPluginName, handlerCreator, serverCreator, helpProvider)
 }
 
 func handlerCreator(githubClient *gogh.Client) server.GitHubEventHandler {
-	return &GitHubLabelsEventsHandler{
-		Client: githubClient,
-	}
+	return &GitHubLabelsEventsHandler{Client: githubClient}
 }
 
 func serverCreator(webhookSecret []byte, eventHandler server.GitHubEventHandler) *server.Server {
 	return &server.Server{
 		GitHubEventHandler: eventHandler,
 		HmacSecret:         webhookSecret,
-		Log:                log,
 	}
 }
 
 // HandleEvent is an entry point for the plugin logic. This method is invoked by the Server when
 // events are dispatched from the /hook service
-func (gh *GitHubLabelsEventsHandler) HandleEvent(eventType github.EventType, eventGUID string, payload []byte) error {
-	gh.log = logrus.StandardLogger().WithField("ike-plugins", ProwPluginName).WithFields(
-		logrus.Fields{
-			"event-type": eventType,
-			"event-GUID": eventGUID,
-		},
-	)
-
-	gh.log.Info("Handling labels event.")
+func (gh *GitHubLabelsEventsHandler) HandleEvent(log log.Logger, eventType github.EventType, payload []byte) error {
+
```

Comment:
is doing nothing intended?

Reply:
not sure we need to log it every time we receive event. we have it tested so we are only really interested in errors we should react on I think

Reply:
ah, ok you mean empty method... well... we haven't started working on this plugin yet :)